src/login.js

```diff
@@ -226,10 +240,20 @@ const OAuthTab = ({
   selectedAuthConfig,
   setSelectedAuthConfig,
   authConfigs,
+  selectedProfile,
+  setSelectedProfile,
+  profiles,
+  profilesState,
 }) => {
   const theme = useTheme()
+  const [missingProfileSelection, setMissingProfileSelection] = React.useState(false)
 
   const login = async (authConfig) => {
+    if (!selectedProfile) {
+      setMissingProfileSelection(true)
+      return
```

Comment:
why shortcut login if no profile is selected? shouldn't there be a default selection?